### PR TITLE
LPS-145927

### PR DESCRIPTION
**Pull request by marianoalvarosaiz**

### CODE REVIEW

File: portal-impl/test/unit/com/liferay/portal/util/LocalizationImplUnitTest.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ public Object invoke(
 
 		localizationUtil.setLocalization(new LocalizationImpl());
 
-		Locale contentDefaultLocale = LocaleUtil.fromLanguageId("es_ES");
+		Locale contentDefaultLocale = LocaleUtil.fromLanguageId("es_ES", false);
```
Comment (tinatian) [inline portal-impl/test/unit/com/liferay/portal/util/LocalizationImplUnitTest.java:154]: @marianoalvarosaiz Why you made this change? I do not understand why this is related. I revert this change and LocalizationImplUnitTest passed.

Tina

Comment (marianoalvarosaiz) [reply portal-impl/test/unit/com/liferay/portal/util/LocalizationImplUnitTest.java:154]: @tinatian it fails without it, see https://github.com/marianoalvarosaiz/liferay-portal/pull/712#issuecomment-1035188861

Anyway, it failed before this LPS also depending on the tests execution order. If you revert the whole LPS and `@Ignore` for example all tests but _testGetDefaultImportLocale3_ and execute it it will fail.

Before the LPS it failed because _es_ES_ wasn't added to the locales map unless testGetDefaultImportLocale1, that assumes _es_ES_ as an available locale, was executed before. So _es_ES_ could not be validated as an available locale unless _testGetDefaultImportLocale1_ or _testGetDefaultImportLocale5_ are executed on first place.

After the LPS it will because _es_ES_ in _testGetDefaultImportLocale3_ is not an available locale, and it will never be validated as so. Then we need to retrieve the locale without validation.

**Comment by tinatian** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/357

**Comment by tinatian** [comment]
ci:close
